### PR TITLE
Term Entry Lua Math cos()

### LINKED ISSUE #3390: [Term Entry] Lua Math .cos()
### Reason/inspiration (optional)

We would like a new term entry on the `.cos()` method for Lua: [Math](https://www.codecademy.com/resources/docs/lua/mathematical-library). The entry should go in a new file under `docs/content/lua/concepts/mathematical-library/terms/cos/cos.md`.

The entry should describe what the method does. Be sure to include the following sections:

- A "Syntax" section with a ` ```pseudo` code snippet.
- An "Example" section with a ` ```lua `code snippet.

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by LewisMK**
### Description

Term entry for Lua math .cos() function. Closes #3390

### Type of Change

- Adding a new entry

### Checklist

- [*] All writings are my own.
- [*] My entry follows the Codecademy Docs style guide.
- [*] My changes generate no new warnings.
- [*] I have performed a self-review of my own writing and code.
- [*] I have checked my entry and corrected any misspellings.
- [*] I have made corresponding changes to the documentation if needed.
- [*] I have confirmed my changes are not being pushed from my forked `main` branch.
- [*] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [*] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/lua/concepts/mathematical-library/terms/cos/cos.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+---
+Title: 'cos()'
+Description: 'Evaluates the cosine of an angle represented in radians.'
```
Comment (caupolicandiaz) [inline content/lua/concepts/mathematical-library/terms/cos/cos.md:3]:
```suggestion
Description: 'Returns the cosine of an angle represented in radians.'
```

File: content/lua/concepts/mathematical-library/terms/cos/cos.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  - 'Functions'
+  - 'Math'
+  - 'Numbers'
+  - 'Arithmetic'
+CatalogContent:
+  - 'learn-lua'
+  - 'paths/computer-science'
+---
+
+The **`cos()`** function evaluates the cosine of an angle in radians.
```
Comment (caupolicandiaz) [inline content/lua/concepts/mathematical-library/terms/cos/cos.md:16]:
```suggestion
The **`cos()`** function returns the cosine of an angle given in radians.
```

File: content/lua/concepts/mathematical-library/terms/cos/cos.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+## Syntax
+
+```pseudo
+$result = math.cos(angle);
+```
+
+`result`: The variable to store the output of the cosine operation.
+`math`: The standard Lua math library.
+`cos()`: The cosine function.
```
Comment (caupolicandiaz) [inline content/lua/concepts/mathematical-library/terms/cos/cos.md:26]:
```suggestion
```

File: content/lua/concepts/mathematical-library/terms/cos/cos.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+`angle`: The angle in radians for which you want to calculate the cosine.
```
Comment (caupolicandiaz) [inline content/lua/concepts/mathematical-library/terms/cos/cos.md:27]:
```suggestion
`angle`: The angle in radians.
```

File: content/lua/concepts/mathematical-library/terms/cos/cos.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+print(cosineOutput)
+```
+
+This example will result in the following output:
+
+```shell
+0.70710678118655
+```
+
+## Codebyte Example
```
Comment (caupolicandiaz) [inline content/lua/concepts/mathematical-library/terms/cos/cos.md:46]: the Codecademy compiler doesn't currently support lua. Please make this a standard example or delete it

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@LewisMK I've added my initial comments. Please revise, and tag me when you've finished making changes. Thanks

File: content/lua/concepts/mathematical-library/terms/cos/cos.md (hunk 5 above)
Comment (LewisMK) [reply content/lua/concepts/mathematical-library/terms/cos/cos.md:44]: Thank you for the edits @caupolicandiaz . What do you mean by a standard example..?

Comment (caupolicandiaz) [reply content/lua/concepts/mathematical-library/terms/cos/cos.md:44]: An example that is not a codebyte, which will include a separate ```shell block with the output

**Review by caupolicandiaz — APPROVED** [review]
@LewisMK I've gone ahead and made the missing changes. This is ready for a second review.

**Review by KTom101 — APPROVED** [review]
@LewisMK Made a minor change to the meta-data. Otherwise, it's good to go. Thanks!

**Pull request merged**